source/upgrading/upgrade/kvm-417.rst

```diff
@@ -0,0 +1,30 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information#
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+KVM agents CPU speed detection issue
+====================================
+
+There is an issue with AMD (and possibly other CPU architectures) on the CPU speed detection at the 
+4.17.0 and 4.17.0.1 CloudStack agents on KVM hosts at startup. The CloudStack agents will attempt to 
+get the CPU speed  by reading the `/sys/devices/system/cpu/cpu0/cpufreq/base_frequency` information. 
```

Comment:
cloudstack agent tries to get cpu speed in different ways
see https://github.com/apache/cloudstack/pull/5506

I personally think it is a libvirt issue, `virsh nodeinfo` returns incorrect value with AMD cpus.

see https://github.com/apache/cloudstack/issues/6680#issuecomment-1234492354